importer/helper.go

```diff
@@ -280,6 +276,16 @@ func checkGenesisHash(db idb.IndexerDb, genesisReader io.Reader) error {
 	if err != nil {
 		return fmt.Errorf("error decoding genesis, %w", err)
 	}
+	network, err := db.GetNetworkState()
+	if err == idb.ErrorNotInitialized {
```

Comment:
If you use `errors.Is(err, idb.ErrorNotInitialized)` here, you can keep the wrapped error message above.